src/service/util/team.ts

```diff
@@ -26,3 +27,13 @@ export function teamFromControl(control: number): number {
         return 0;
     }
 }
+
+export function isRoundOver(world: CountryData[]): boolean {
+    const axisSpanws = world.filter(
+        (c) => c.isActiveSpawn && c.isSpawn && c.initialControl === 0
+    );
+    const alliesSpanws = world.filter(
+        (c) => c.isActiveSpawn && c.isSpawn && c.initialControl === 100
+    );
+    return alliesSpanws.length === 0 || axisSpanws.length === 0;
```

Comment:
nitpicky, but `some` would be better suited for this than `filter` + `length`

Reply:
changed